README.md

```diff
@@ -19,6 +19,18 @@ Then you will create an instance of imported class, by choosing one of the follo
     ```
     var factory = CommonFactory.createFromArguments(args);
     ```
+1. Create factory with a namespace in Kubernetes and the name of the target th2 box from Kubernetes:
+    ```
+    var factory = CommonFactory.createFromKubernetes(namespace, boxName);
+    ```
+    
+### Requirements for creatring factory with Kubernetes
+
+1. It is necessary to have Kubernetes configuration written in ~/.kube/config. 
```

Comment:
Could you please add the link to the page where `Kubernetes configuration` is described